src/components/Combobox/Combobox.stories.tsx

```diff
@@ -20,104 +20,49 @@ const meta: Meta<typeof Combobox> = {
     options,
     label: "Pick a color",
     id: "combobox",
-    value: "color-black",
+    size: "large",
   },
 };
 
 export default meta;
 type Story = StoryObj<typeof Combobox>;
 
-export const Large: Story = {
-  args: {
-    size: "large",
+const ComboboxTemplate: Story = {
+  render: (args) => {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const [value, setValue] = useState(options[0].value);
+
+    return (
+      <Combobox
+        {...args}
+        value={value}
+        onChange={(value) => setValue(value as string)}
+      />
+    );
```

Comment:
issue: this causes two issues: firstly wrong rendering of your docs (look at generated snippets), secondly it confises users because they see working component with code to copy-paste while it actually requires state to handle. Consider using custom snippets: https://github.com/saleor/saleor-dashboard/pull/3586